### PR TITLE
fix(position): fix scroll jank caused by the position  component

### PR DESCRIPTION
**Pull request by EdwardIrby**
https://pluralsight.slack.com/archives/C70HPSJPP/p1632416018084000

### CODE REVIEW

File: packages/position/src/react/index.tsx [hunk 1]
```diff
@@ -160,7 +160,10 @@ export const Position = React.forwardRef<HTMLElement, PositionProps>(
             )}
 
         {createUniversalPortal(
-          <div style={{ visibility: shownOnce ? 'visible' : 'hidden' }}>
+          <div
+            style={{ display: shownOnce ? 'initial' : 'none' }}
```
Comment (danethurber) [inline packages/position/src/react/index.tsx:164]: 👍  guessing this is to hide the scrollbar. was changing the overflow an option at all

File: packages/position/src/react/index.tsx [hunk 2]
```diff
@@ -160,7 +160,10 @@ export const Position = React.forwardRef<HTMLElement, PositionProps>(
...
+            aria-live="polite"
```
Comment (danethurber) [inline packages/position/src/react/index.tsx:165]: is this to limit the amount of updates read out to the user while scrolling?

**Review by danethurber — APPROVED** [review]
LGTM. I asked a couple questions for my own edification

File: packages/position/src/react/index.tsx (hunk 2 above)
Comment (EdwardIrby) [reply packages/position/src/react/index.tsx:165]: I wanted to let the screen reader know something will eventually appear here after a user interaction. Initially on mount this is an empty div. We should let assistive technology know its dynamic and may render content at some point (politely).

File: packages/position/src/react/index.tsx (hunk 1 above)
Comment (EdwardIrby) [reply packages/position/src/react/index.tsx:164]: Overflow not really. Other than that single style this `div` had no other styles applied to it. It's just the mount point for a positioned element. We do positioning on the `showEl`  we're cloning. I could probably try add some checks to make sure `showEl` is rendering on screen the first pass but that's more JS for a simple css fix.

**Pull request merged**